app/models/concerns/taggable.rb

```diff
@@ -0,0 +1,17 @@
+module Taggable
```

Comment:
Gotcha. So the path goes:

Tag added --> Record Saved --> after_commit triggers indexing --> Tags "touches" Record --> Record sets `cached_tags` from Tags table --> after_commit triggers indexing --> as_indexed_json overrides `tags` field and removes `cached_tags`.

Is that right? I might be oversimplifying this, but would it be possible to skip the `cached_tags` entirely and instead just use `tag_list` in the [`as_indexed_json`](https://github.com/greencommons/commons/pull/120/files#diff-13edb83250a6faff4a1518a8dee4f9a2R14) override? Either way the record gets indexed twice, right? Maybe I'm thinking of the order of operations incorrectly...

Seems pretty complicated, but ES forces us to go this way I suppose. Thanks for working through this.